src/cluster/index.js

```diff
@@ -218,6 +218,11 @@ module.exports = class Cluster {
     }
 
     const topicMetadata = metadata.topicMetadata.find(t => t.topic === topic)
+    if (!topicMetadata) {
+      throw new KafkaJSTopicMetadataNotLoaded('Topic metadata does not include required topic', {
```

Comment:
A note here: This is an API change, in the sense that a partition assigner would typically call this function. 

It is possible that members of a group see different topics, and the assigner cannot just assume that it can look up the cached metadata for all topics known to all members in the group.
Still I think this is fair, and the assigner code should actually explicitly handle this error. In our case for example I know that the issue is transient and will go away with one of the next runs, so we simply ignore it as before.

Reply:
In testing it showed that while this might be "theoretically nice", the change leads to a lot of places that now need to handle the error, but sometimes the error isn't actually an error.

Dropped this one again for that reason.